recipes/recipe/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "automan" %}
+{% set version = "0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2ef29e1f7d6133af9c428ab8e549da2101bccdd0cb6fb9a2caa48c70af8de226
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - automan
+
+about:
+  home: https://github.com/pypr/automan
+  license: MIT
+  license_file: LICENSE.txt
+  summary: 'automan'
+
+
+extra:
+  recipe-maintainers:
+    - prabhuramachandran
```

Comment:
@prabhuramachandran Are you OK with being a maintainer here?

Reply:
Yes, that should be OK.  For the time being @rishi9699 is going to setup the recipes and make sure that it works correctly.